.github/workflows/test_and_build.yml

```diff
@@ -152,16 +152,17 @@ jobs:
       - run: zig/zig build fuzz_vsr_superblock_quorums -- --seed 123
 
   # This is just a canary to make sure that the simulator compiles
-  # (and passes for at least a single seed).
-  # simulator:
-  #   name: 'Simulator canary'
-  #   runs-on: ubuntu-latest
-  #   matrix:
-  #     state_machine: [accounting, test]
-  #   steps:
-  #     - uses: actions/checkout@v2
-  #     - run: ./scripts/install_zig.sh
-  #     - run: zig/zig run -OReleaseSafe -Dconfig-cluster-state-machine=${{matrix.state_machine}} src/simulator.zig -- 123
+  # It would be a good idea to also _run_ a single iteration,
+  # but that currently has some false failures :-)
+  simulator:
```

Comment:
I think `simulator` needs to be added to `bors_tigerbeetle`'s dependencies.

Reply:
Ah, that's annoying, thanks, fixed!